functional-tests/CloudbaseInit.Tests.ps1

```diff
@@ -287,6 +301,29 @@ function after.cloudbaseinit.plugins.windows.bootconfig.BootStatusPolicyPlugin {
 
 function prepare.nocloud {
     pushd "$here/../$($env:CLOUD)"
+
+        $openVpnUrl = "https://build.openvpn.net/downloads/releases/OpenVPN-2.5.10-I601-amd64.msi"
+        $wc = New-Object System.Net.WebClient
+        $msiFilePath = Join-Path $(pwd) "openvpn.msi"
+        $wc.DownloadFile($openVpnUrl, $msiFilePath)
+        cmd /c "msiexec.exe -i ${msiFilePath} /qn /norestart /l*v test.log"
+        Start-Sleep 10
+        if ($LASTEXITCODE) { throw "Failed to install openvpn" }
+        Get-NetAdapter | Out-Null
+        Start-Sleep 10
+        $adapter = Get-NetAdapter -Physical | Select-Object -First 1
+        $adapter = Get-NetAdapter -InterfaceDescription "TAP-Windows Adapter V9" | Select-Object -First 1

```

Comment:
We could add a comment stating that "A tap device gets created by default after installing OpenVPN. We'll update the network-config template with the according MAC address."